test-dev/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+if(POLICY CMP0079)
+    cmake_policy(SET CMP0079 NEW)
+endif()
+
+if(POLICY CMP0115)
+    cmake_policy(SET CMP0115 NEW)
+endif()
+
+include_directories(
+    ${CMAKE_SOURCE_DIR}/include
+    ${CMAKE_SOURCE_DIR}/src)
+
+set(DEVTEST_SRC
+    util.c
+    main.c
+    simple_module.c
+    compare_mixer_data.c
+    ../src/md5.c
+    ../src/filetype.c
+    ../src/hio.c
+    ../src/load_helpers.c
+    ../src/period.c
+    ../src/memio.c
+    ../src/dataio.c
+    ../src/scan.c
+    ../src/loaders/itsex.c
+    ../src/loaders/sample.c
+    ../src/loaders/common.c
+    ../src/depackers/xfnmatch.c
+# AMIGA
+    ../src/depackers/xfd.c
+    ../src/depackers/xfd_link.c
+# WIN32 OR WATCOM
+    ../src/win32.c
+    ../src/depackers/ptpopen.c
```

Comment:
The autotools Makefile.in pulls in only `win32.c`, but not amiga `xfd.c` and `xfd_link.c`
nor windows `ptpopen.c`: you shouldn't need those three.

Reply:
So, I'll put these files into the platform specific appends

Reply:
Fun fact: at my initial build they WERE in the own branches, however, you told me to merge them with lists and don't separate them by if(OS)... .

Reply:
Your commit 82c4e0d5f1 is wrong: revert it.

I am talking about `test-dev/CMakeLists.txt`